web/src/components/AssertionCard/AssertionCardSelectorList.tsx

```diff
@@ -1,14 +1,29 @@
 import OperatorService from '../../services/Operator.service';
 import {TPseudoSelector, TSpanSelector} from '../../types/Assertion.types';
+import {TViewResultsMode} from '../../types/TestDefinition.types';
 import * as S from './AssertionCard.styled';
 
 interface IProps {
   selectorList: TSpanSelector[];
   pseudoSelector?: TPseudoSelector;
+  isAdvancedSelector: boolean;
+  selector: string;
+  viewResultsMode: TViewResultsMode;
 }
 
-const AssertionCardSelectorList: React.FC<IProps> = ({selectorList, pseudoSelector}) => {
-  return (
+const AssertionCardSelectorList = ({
+  selectorList,
+  pseudoSelector,
+  isAdvancedSelector,
+  selector,
+  viewResultsMode,
+}: IProps) => {
+  return isAdvancedSelector || viewResultsMode === 'advanced' ? (
```

Comment:
The view results mode + the advanced selector would help us decide how to show the info